iceberg/pom.xml

```diff
@@ -26,7 +26,7 @@
     <hive.path.to.root>..</hive.path.to.root>
     <path.to.iceberg.root>.</path.to.iceberg.root>
     <iceberg.version>1.3.0</iceberg.version>
-    <kryo-shaded.version>4.0.3</kryo-shaded.version>
+    <kryo.version>5.0.3</kryo.version>
     <iceberg.mockito-core.version>3.4.4</iceberg.mockito-core.version>
     <iceberg.avro.version>1.11.1</iceberg.avro.version>
     <iceberg.kryo.version>5.2.0</iceberg.kryo.version>
```

Comment:
@devaspatikrishnatri We already have iceberg.kryo.version here set to 5.2.0,  Why do we need another kryo.version with a lower value